services/supabase/gitauto_manager.py

```diff
@@ -137,6 +154,16 @@ def create_user_request(
             )
             .execute()
         )
+        
+        if not str(email).lower().endswith("@users.noreply.github.com"):
```

Comment:
Should this condition be handled in get_user_public_email()?

Reply:
well get_user_public_email is simply for retrieving the email, but i agree that we can consolidate it somehow